src/DelimitedDataParser/Exporter.cs

```diff
@@ -30,11 +30,17 @@ public class Exporter
         private bool _useExtendedPropertyForColumnName;
         private string _extendedPropertyKey;
 
+        private IProgress<int> _progress;
```

Comment:
This should be declared as `readonly`, and placed above non-`readonly` properties.